lib/yabeda/histogram.rb

```diff
@@ -20,7 +20,7 @@ def measure(tags, value = nil)
 
       all_tags = ::Yabeda::Tags.build(tags, group)
       values[all_tags] = value
-      ::Yabeda.adapters.each do |_, adapter|
+      ::Yabeda.adapters.each_value do |adapter|
```

Comment:
Shouldn't adapters also be filtered here? (and in other metric types)

Reply:
Yes, you are right! Adapters should be filtered